src/sql/workbench/parts/query/browser/queryEditor.ts

```diff
@@ -294,7 +294,8 @@ export class QueryEditor extends BaseEditor {
 	 * Sets this editor and the 2 sub-editors to visible.
 	 */
 	public setEditorVisible(visible: boolean, group: IEditorGroup): void {
-		this.currentTextEditor.setVisible(visible, group);
+		this.textFileEditor.setVisible(visible, group);
```

Comment:
So this works because despite both being "visible" we still only have one added to the DOM at a time? Seems a little odd - I suppose this is mostly because of our weird dual editor thing and so is something that the native vs code editors don't need a special case for?